.husky/pre-commit

```diff
@@ -1,4 +1,24 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn run lint-staged
+yarn run lint
```

Comment:
`lint-staged` is only linting the files about to be commited, whereas `lint` will be linting on all files in the repository.

Are we ok with the widening of this check?

Reply:
@merlinc This must have been a mistake as I don't remember changing this.